README.md

```diff
@@ -11,3 +11,84 @@ Modes:
 Seeing how far you can get going level by level like pacman
 
 Win level by level each unlocking the next or some mutation of that
+
+# Proposal (04/03/2018)
+
+## Team
+I will be working individually for this project and Satjit Bola will be acting as my code reviewer.
+
+
+## Features
+
+### Game Summary:
+
+An Arkit app where 3D UFOs perform different maneuvers while shooting lasers at the user. The user can Fireback and dodge the enemy lasers. Wave after wave of enemies (Mostly 3D UFOs) come until the Player takes too many hits and the game is over.
+
+During gameplay the Player gets points based on several factors; Time spent Playing, number of Enemy UFOs hit by player Lasers, and more.
```

Comment:
Consider changing these to bullet points as you've done at later parts in this proposal - for consistency